Jenkinsfile

```diff
@@ -94,9 +94,9 @@ pipeline {
                   pmdParser(
                     pattern: '**/build/reports/pmd/*.xml'
                   ),
-                  spotBugs(
-                    pattern: '**/build/reports/findbugs/*.xml'
-                  )
+                  //spotBugs(
+                  //  pattern: '**/build/reports/findbugs/*.xml'
+                  //)
```

Comment:
Do we need a ticket to revisit this part or can this be removed already?

Reply:
Very good idea. I created a ticket KIEKER-1854 to fix and re-enable spotbugs.